### PR TITLE
Include CCX courses in org exports

### PR DESCRIPTION
**Pull request by HassanJaveed84**
@mulby @brianhw

### CODE REVIEW

File: exporter/tasks.py [hunk 1]
```diff
@@ -54,10 +54,15 @@ class CourseTask(object):
     def get_filename(cls, **kwargs):
         course_key = CourseKey.from_string(kwargs['course'])
 
+        if hasattr(course_key, 'ccx'):
+            course = '-'.join((course_key.org, course_key.course, course_key.run, 'ccx', course_key.ccx))
+        else:
+            course = '-'.join((course_key.org, course_key.course, course_key.run))
+
```
Comment (brianhw) [inline exporter/tasks.py:76]: This logic appears twice -- can we pull it out into a method?

**Comment by brianhw** [comment]
Just the one issue.

File: exporter/tasks.py (hunk 1 above)
Comment (HassanJaveed84) [inline exporter/tasks.py:76]: Moved it to a class method, should I move it to util.py ?

Comment (brianhw) [inline exporter/tasks.py:76]: Nah, I think it's fine here, since it's only needed here.

**Comment by HassanJaveed84** [comment]
Should I merge this ?

**Comment by mulby** [comment]
:+1: from me

**Comment by brianhw** [comment]
And :+1:  from me....

**Pull request merged**